.github/workflows/CI.yml

```diff
@@ -36,25 +45,28 @@ jobs:
     - name: Install RF ${{ matrix.rf-version }}
       run: |
         pip install -U --pre robotframework==${{ matrix.rf-version }}
+    - name: Install drivers via selenium-manager
+      run: |
+        SELENIUM_MANAGER_EXE=$(python -c 'from selenium.webdriver.common.selenium_manager import SeleniumManager; sm=SeleniumManager(); print(f"{str(sm.get_binary())}")')
+        echo "$SELENIUM_MANAGER_EXE"
+        echo "WEBDRIVERPATH=$($SELENIUM_MANAGER_EXE --browser chrome --debug | awk '/INFO[[:space:]]Driver path:/ {print $NF;exit}')" >> "$GITHUB_ENV"
+        echo "$WEBDRIVERPATH"
     - name: Generate stub file for ${{ matrix.python-version }}
-      if: matrix.python-version != 'pypy-3.7'
+      if: matrix.python-version != 'pypy-3.9'
```

Comment:
same here
` # Recognize for the moment this will NOT run as we aren't using CPython 3.9`